crates/validation/src/capture.rs

```diff
@@ -389,6 +393,19 @@ async fn walk_capture<C: Connectors>(
     }
     let inactive_bindings = live_bindings_spec.values().map(|v| (*v).clone()).collect();
 
+    // Use manual salt if provided, otherwise the live salt, otherwise generate a new one.
```

Comment:
What do you think about requiring a backfill if the salt changes? Kinda seems like if you care enough to set the salt manually, then you'd want to backfill if it changes. Definitely doesn't seem necessary, especially in the short term, though.

Reply:
It's a fairly advanced feature so I don't have a handle on when or why people will use it. Rotation? Alignment with another capture? Matching a migrated table ? :shrug: